docs/modules/language-reference/pages/index.adoc

```diff
@@ -685,7 +685,7 @@ dodo {
 As you probably guessed, the nested property `class` can be accessed with `dodo.taxonomy.class`.
 
 Like all values, objects are _immutable_, which is just a fancy (and short!) way to say that their properties never change.
-So what happens when Pigeon moves to a different street? Do we have to construct a new object from scratch?
+So what happens when dodo moves to a different street? Do we have to construct a new object from scratch?
```

Comment:
```suggestion
So what happens when Dodo moves to a different street? Do we have to construct a new object from scratch?
```

(if referring to the _property_ it should be between backticks, but this previously spoke casually about Pigeon, referring to the given `name`).